Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -709,70 +726,124 @@ void VolumeVerifier::Process()
   else if (m_block_index < m_blocks.size() && m_blocks[m_block_index].offset == m_progress)
   {
     bytes_to_read = VolumeWii::BLOCK_TOTAL_SIZE;
+    block_read = true;
   }
   else if (m_block_index < m_blocks.size() && m_blocks[m_block_index].offset > m_progress)
   {
     bytes_to_read = std::min(bytes_to_read, m_blocks[m_block_index].offset - m_progress);
   }
   bytes_to_read = std::min(bytes_to_read, m_max_progress - m_progress);
 
+  bool read_succeeded = false;
+  std::vector<u8> data(bytes_to_read);
+  if (m_calculating_any_hash || content_read || block_read)
+  {
+    std::lock_guard<std::mutex> lk(m_volume_mutex);
```

Comment:
Starting with C++17, the mutex type can be omitted like:

```suggestion
    std::lock_guard lk(m_volume_mutex);
```

Due to deduction guides being introduced